sdk/src/gateway/abi.ts

```diff
@@ -83,3 +83,48 @@ export const offRampCaller = [
         stateMutability: 'nonpayable',
     },
 ] as const;
+
+export const compoundV2CTokenAbi = [
```

Comment:
Not blocking on this PR, but I wonder if ethers-js has some macro to generate these from a function definition?

Reply:
Yes exactly!